src/bzfs/commands.cxx

```diff
@@ -2805,8 +2812,14 @@ bool ReloadCommand::operator() (const char   *message,
     }
     else
     {
+        // Allow a plug-in to take over
```

Comment:
Assuming the unconditional event manager call is correct, then shouldn't this be moved immediately after it?

I apologize for not being clear on how the event manager interacts; is the point to "do what we do" and allow plugins to extend, or do we allow plugins to override?